snowpack/src/util.ts

```diff
@@ -409,3 +409,5 @@ export function removeLeadingSlash(path: string) {
 export function removeTrailingSlash(path: string) {
   return path.replace(/[/\\]+$/, '');
 }
+
+export const hmrFilePath = path.join(__dirname, '../assets/hmr.js')
```

Comment:
since in both cases we use the file contents, can we instead do:

```
// currently in dev.ts
const HMR_DEV_CODE = fs.readFileSync(path.join(__dirname, '../assets/hmr.js'));
```

I'd rather keep the implementation detail (the fact that we're copying from an internal file) hidden from the rest of the codebase, in case we want to change this in the future.

Reply:
nice, I modified.